react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -1,108 +1,86 @@
 import { useState } from 'react';
 
-const languages = [
-  {
-    name: 'Hypertext Markup Language',
-    text: `The HyperText Markup Language or HTML is the standard markup language
-        for documents designed to be displayed in a web browser. It is often
-        assisted by technologies such as Cascading Style Sheets (CSS) and
-        scripting languages such as JavaScript.`,
-  },
-  {
-    name: 'Cascading Style Sheets',
-    text: `Cascading Style Sheets (CSS) is a style sheet language used for
-        describing the presentation of a document written in a markup language
-        such as HTML or XML (including XML dialects such as SVG, MathML or
-        XHTML).[1] CSS is a cornerstone technology of the World Wide Web,
-        alongside HTML and JavaScript.`,
-  },
-  {
-    name: 'JaveScript',
-    text: `JavaScript (JavaScript, often abbreviated as JS, is a programming
-        language that is one of the core technologies of the World Wide Web,
-        alongside HTML and CSS. As of 2022, 98% of websites use JavaScript on
-        the client side for webpage behavior, often incorporating third-party
-        libraries. All major web browsers have a dedicated JavaScript engine
-        to execute the code on users' devices.`,
-  },
-];
 /**
  * function that returns an accordion menu
  * @returns a JSX element
  */
-export default function Accordion(): JSX.Element {
+
+type PropsAccordion = {
+  languages: { name: string; text: string }[];
```

Comment:
Does your accordion only display languages? ;-) Maybe this should be named "items"?